event_email_reminder/data/event_email_reminder_data.xml

```diff
@@ -1,52 +1,52 @@
         <field
             name="email_from"
-        >${object.company_id.name}&lt;${object.company_id.email}&gt;</field>
+        >{{object.company_id.name}}&lt;{{object.company_id.email}}&gt;</field>
         <field name="subject">The events will be started soon</field>
-        <field name="email_to">${object.email|safe}</field>
-        <field name="lang">${object.lang}</field>
+        <field name="email_to">{{object.email}}</field>
+        <field name="lang">{{object.lang}}</field>
         <field name="model_id" ref="base.model_res_partner" />
         <field name="auto_delete" eval="True" />
-        <field name="body_html">
-        <![CDATA[
-<p>Hello ${object.name},&nbsp;</p>
-<p>This is a event's summary: </p>
-<table>
-    <thead>
-        <tr style="height: 15px; background-color: #d8d8d8;font-size: 13px;font-weight:bold;" align="center" valign="middle">
-            <td>Date Begin</td>
-            <td>Event Name</td>
-            <td>Date End</td>
-            <td>State</td>
-        </tr>
-    </thead>
-    % for event in ctx["events"]:
-    <tr>
-        <td><span>${event.date_begin}</span></td>
-        <td><span>${event.name}</span></td>
-        <td><span>${event.date_end}</td>
-        <td><span style="font-weight:bold;">${event.state}</td>
-    </tr>
-    % endfor
-</table>
-<pre>
---
-Automatic Email.
-${object.company_id.name or ''}
-</pre>]]>
+        <field name="body_html" type="html">
+            <p>Hello <t t-out="object.name" />,</p>
+            <p>This is a event's summary: </p>
+            <table>
+                <thead>
+                    <tr
+                        style="height: 15px; background-color: #d8d8d8;font-size: 13px;font-weight:bold;"
+                        align="center"
+                        valign="middle"
+                    >
+                        <td>Date Begin</td>
+                        <td>Event Name</td>
+                        <td>Date End</td>
+                        <td>State</td>
+                    </tr>
+                </thead>
+                <tr t-foreach="ctx['events']" t-as="event">
+                    <td><span t-out="event.date_begin" /></td>
+                    <td><span t-out="event.name" /></td>
+                    <td><span t-out="event.date_end" /></td>
+                    <td><span
+                            t-out="event.stage_id.name"
+                            style="font-weight:bold;"
+                        /></td>
+                </tr>
+        </table>
     </field>
     </record>
     <record id="ir_cron_event_email_reminder_action" model="ir.cron">
         <field name="name">Event Email Reminder</field>
         <field name="active" eval="True" />
         <field name="model_id" ref="event.model_event_event" />
-        <field name="code">model.run_event_email_reminder(7, False, False, None)</field>
+        <field name="code">model.run_event_email_reminder(7, False, None)</field>
```

Comment:
You can separate this file by model: `ir_cron_data.xml` and `mail_template_data.xml`. About this cron, we are changing the behavior of the module: on previous version, "new" events were not included. Now, all the events are included. The change on the data model has been to have a flexible stage system, through a new model `event.stage`. I think you should include a flag in it to exclude or not from these notifications, and pre-mark as excluded the stage "New".

Reply:
Ok, changes done.